EpiserverRedirects/Mapper/IRedirectRuleMapper.cs

```diff
@@ -4,7 +4,7 @@ namespace Forte.EpiserverRedirects.Mapper
 {
     public interface IRedirectRuleMapper
```

Comment:
Let's maybe emphasize that it's `Dto` mapper? We already have `IRedirectRuleMapper` in solution (which is in EF project but as I said in different comment I would vote for making it generic and reuse for DDS mapper as well) so it's a little bit confusing when searching for a particular file/class